File: DemoApplication.Web/src/app/models/auth/new-maker-checker.model.ts
import { MakerCheckerActionsEnum } from "./../enums.model";
// import { FileModel } from "../file/file.model";

export class NewMakerCheckerModel {
    constructor(
        public action?: MakerCheckerActionsEnum,
        public motivation?: string,
        public files?: string[],
        public model?: string
    ) {
    }
}

File: DemoApplication.Models/MakerChecker/NewMakerCheckerModel.cs
using System;
using DemoApplication.Enums;

namespace DemoApplication.Models;

public class NewMakerCheckerModel
{
    public Guid Id { get; set; }
    public MakerCheckerActionsEnum Action { get; set; }
    public string Motivation { get; set; } = string.Empty;
    public Guid[] Files { get; set; } = Array.Empty<Guid>();
    public string Model { get; set; } = string.Empty;
}
